README.md

```diff
@@ -1,16 +1,15 @@
-# cloud-report-2021
+# cloud-report-2022
 
-Details about and from the Cloud Report done in Q4 2020
+Details about and from the Cloud Report done in Q2 2022
 
-In 2018, we produced a "Cloud Report", wherein we assessed the performance of AWS vs. GCP along dimensions like TPC-C throughput, I/O, CPU and latency. This year, we aim to produce a similar report but including Azure.
+From 2018 to 2021, we produced the series of "Cloud Report", wherein we assessed the performance of AWS vs. GCP along dimensions like TPC-C throughput, I/O, CPU and latency.
 
 This repo will let us aggregate and share data among team members, including processes and results. 
 
+Resources for Cloud Report last year (2021)
 * [Highlights from the 2021 Cloud Report](https://www.cockroachlabs.com/blog/2021-cloud-report/)
 * [2021 Cloud Report](https://www.cockroachlabs.com/guides/2021-cloud-report)
```

Comment:
Should we add the placeholders for the 2022 urls; I imagine they would be the same modulo `2022`.